Client/overrides/config/ftbquests/classic/chapters/3bc6db27/1f1dad8c.snbt

```diff
@@ -2,6 +2,7 @@
 	title: "§6Infinity Drill [Legendary]",
 	x: -3.5d,
 	y: -7.0d,
+	description: "To complete the quest change the drill mode to Legendary with Shift+Right Click.",
```

Comment:
```suggestion
	description: "To complete the quest change the drill mode to Legendary with Shift+Right Click.",
```
I'd say change it to "Change the drill to Legendary mode to complete the quest (Shift+Right-click to change modes)"